streamer/bitrate_configuration.py

```diff
@@ -154,6 +155,7 @@ def _sortable_properties(self) -> Tuple[float]:
       'opus': '32k',
       'ac3': '96k',
       'eac3': '48k',
+      'flac': '705k',
```

Comment:
I agree to review these values ​​if you consider it.

Reply:
Does FFmpeg even honor them?  FLAC is lossless, so we don't really control the bitrate, do we?

Reply:
It doesn't control it, but we need to add something here to make all our logic work.

Reply:
Understood.  Then these numbers are fine.